frontend/packages/console-demo-plugin/src/plugin.tsx

```diff
@@ -187,6 +187,16 @@ const plugin: Plugin<ConsumedExtensions> = [
       },
     },
   },
+  {
```

Comment:
Can you demonstrate this new functionality by migrating the `/terminal` route instead of creating a test route?

Reply:
Nice :+1: 
For this PR I want to have smallest footprint, a simple route show how to use it while adding minimal code ... maybe we can do a variant of the VNC terminal PR that use this plugin option ?

Reply:
Agreed with @glekner - if we're introducing a mechanism to support "full screen" pages (rendered outside the standard Console `<App>` layout) then we should also refactor existing code to utilize this mechanism for better consistency.

Right now, our React router has two top-level routes:

```tsx
<Route path="/terminal" component={CloudShellTab} />
<Route path="/" component={App} />
```

 @yaacov @spadgett @christianvogt Does it make sense to move "cloud shell" (`<CloudShellExec>` / `xterm` based CLI) functionality to one of the existing Console packages?

Reply:
Yes let's convert the cloud shell to use this extension.